README.md

```diff
@@ -312,74 +312,6 @@ public class ModificationsVisitor implements CommitVisitor {
 }
 ```
 
-## Parsing Code
-
-You have the entire source code of the repository. You may want to analyze it. RepoDriller comes with JDT and ANTLR bundled. JDT is the Eclipse internal parser.
-
-Let's say we decide to count the quantity of methods in each modified file. All we have to do is to create a _CommitVisitor_, the way we are used to. This visitor will use our _JDTRunner_ to invoke a JDT visitor (yes, JDT uses visitors as well). Notice that we executed the JDT visitor, and then wrote the result.
-
-```java
-import java.io.ByteArrayInputStream;
-
-import org.repodriller.domain.Commit;
-import org.repodriller.domain.Modification;
-import org.repodriller.parser.jdt.JDTRunner;
-import org.repodriller.persistence.PersistenceMechanism;
-import org.repodriller.scm.CommitVisitor;
-import org.repodriller.scm.SCMRepository;
-
-
-public class JavaParserVisitor implements CommitVisitor {
-
-	@Override
-	public void process(SCMRepository repo, Commit commit, PersistenceMechanism writer) {
-
-		for(Modification m : commit.getModifications()) {
-		
-			if(m.wasDeleted()) continue;
-			
-			NumberOfMethodsVisitor visitor = new NumberOfMethodsVisitor();
-			new JDTRunner().visit(visitor, new ByteArrayInputStream(m.getSourceCode().getBytes()));
-			
-			int methods = visitor.getQty();
-			
-			writer.write(
-				commit.getHash(),
-				m.getFileName(),
-				methods
-			);
-			
-		}
-		
-	}
-}
-```
-
-The visitor is quite simple. It has methods for all different nodes in the file. All you have to do is to visit the right node. As an example, we will visit _MethodDeclaration_ and count the number of times it is invoked (one per each method in this file). 
-
-```java
-import org.eclipse.jdt.core.dom.ASTVisitor;
-import org.eclipse.jdt.core.dom.MethodDeclaration;
-
-public class NumberOfMethodsVisitor extends ASTVisitor {
-
-	private int qty = 0;
-	
-	public boolean visit(MethodDeclaration node) {
-		
-		qty++;
-		return super.visit(node);
-	}
-	
-	public int getQty() {
-		return qty;
-	}
-}
-```
-
-
-If you want to see all methods available, check the documentation for [ASTVisitor](http://help.eclipse.org/juno/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2Fdom%2FASTVisitor.html).
```

Comment:
Do you want to leave links to AST resources in?

Reply:
What if we replace it with a section in the README sketching the kinds of things you can do with RepoDriller (e.g. query the GitHub API, check test coverage, parse and ask questions about the AST, etc.) and put appropriate links there?

Reply:
Yeah, that's the idea. Less esoteric, and more of an impression that RepoDriller 'plays nice' with those things.

Reply:
I extracted the JDT plugin to here: https://github.com/mauricioaniche/repodriller-plugin-jdt. Maybe it requires us to just update JDT dependencies as they evolved since last year.

In addition, I do have another project on my github that I always plug with repodriller when I need to do some JDT magic: https://github.com/mauricioaniche/ck. This ck project uses JDT and calculates lots of code metrics for free; it's also pluggable, meaning that you can add your own metrics if you want.

What do you think? Maybe a word about it on this README? I can do it!

Reply:
I like the idea of a general "Things you can do with RepoDriller" section.

@mauricioaniche I think both of these projects are useful references and should be included in that section. But I agree with @ayaankazerouni that we can be more expansive in our suggestions. This will help new users see the broader possibilities.